src/adapters/adapter.js

```diff
@@ -252,6 +252,15 @@ class Adapter {
   }
 
   /**
+   * Selects the file at a specific path in a new tab window.
+   * @api public
+   */
+  selectFileNewTab(path) {
```

Comment:
These 2 methods do the same thing. Maybe rename to a single one: `openInNewTab(path)`.